benchmarks/README.md

```diff
@@ -63,3 +63,29 @@ To get realistic results, you should exercise care when running benchmarks. Here
 * Blindly believe the numbers that your microbenchmark produces but verify them by measuring e.g. with `-prof perfasm`.
 * Run more threads than your number of CPU cores (in case you run multi-threaded microbenchmarks).
 * Look only at the `Score` column and ignore `Error`. Instead take countermeasures to keep `Error` low / variance explainable.
+
```

Comment:
I wrote all this while I was working on this because I couldn't figure out how to get actual intel assembly instructions. It turns out that if you install the hsdis that they build as part of the openjdk you'll just get jvm opcodes. Those are neat, but not useful to me. Instead you need the hsdis from the "Free Code Manipulation Library". Which you have to build. And it isn't a simple as `git clone foo && cd foo && ./configure && make && sudo make install`. You need to do what I did.

I can separate this out into a separate change if either change is controversial.

Reply:
Seems helpful to me... I would appreciate instructions like this if I were getting started :)